pkg/yqlib/operator_anchors_aliases_test.go

```diff
@@ -283,6 +283,66 @@ var anchorOperatorScenarios = []expressionScenario{
 		expression:     `.thingOne |= explode(.) * {"value": false}`,
 		expected:       []string{expectedUpdatedArrayRef},
 	},
+	{ // Merge anchor with inline map
```

Comment:
put comments as the description instead, they will then be printed on test failure for traceability